nex2art/core/Upload.py

```diff
@@ -398,3 +405,13 @@ def incFileCount(self, fname, error=False):
             self.parent.prog.stepsmap['Artifacts'][4] += 1
             if error: self.parent.prog.stepsmap['Artifacts'][3] += 1
             self.parent.prog.refresh()
+
+    def isNexus3ChecksumFile(self, chapdir, fname):
+        blobpropertiesfile = os.path.join(chapdir, fname + '.properties')
+        blobproperties = open(blobpropertiesfile, 'r').read()
+        return ('.md5' in blobproperties) or ('.sha1' in blobproperties) or ('.sha256' in blobproperties)
```

Comment:
Is this the best way to do it? We don't want to skip artifacts that just happen to have one of these strings somewhere in their metadata. Maybe instead something like:
``` python
blobname = self.acquireMetadata3(blobpropertiesfile)['@BlobStore.blob-name']
return blobname.endswith('.md5') or blobname.endswith('.sha1') or blobname.endswith('.sha256')
```

Reply:
I like this, a much more conservative solution